composer.json

```
@@ -36,7 +36,7 @@
         "twig/twig": "~1.35|~2.4"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "~2.7.5",
+        "friendsofphp/php-cs-fixer": "^2.15.0",
```

```suggestion
        "friendsofphp/php-cs-fixer": "v2.15.0",
```

Better safe then sorry ;)

---

@andrerom how about `~2.15.0` then to get bug fixes?

---

If you like CI suddenly complaining on things it did not before, outside of scope of PR's... :)

---

> If you like CI suddenly complaining on things it did not before, outside of scope of PR changes... :)

As discussed internally a few days ago - for now I've changed it to `~2.15.0` (meaning `>= 2.15.0`, `<2.16.0`). We'll adjust if there would be a breaking change in the future.